### PR TITLE
Added Mailgun email service to tools #2331

### PR DESCRIPTION
**Pull request by NLaff**
Adding Mailgun to the list of email services on the tools page per #2331 

mailgun

note: this is my first PR. I have reviewed the guidelines and signed the CLA.

### CODE REVIEW

File: pages/shared/data/tools.yaml [hunk 1]
```diff
@@ -316,3 +316,13 @@ providers:
       fit: contain
     formats:
       - email
+  - name: Mailgun
+    url: https://mailgun.com/
+    type: Email Service
+    image:
+      src: /static/img/tools/mailgun.png
```
Comment (sebil) [inline pages/shared/data/tools.yaml]: This image doesn't exist yet

File: pages/shared/data/tools.yaml [hunk 2]
```diff
@@ -316,3 +316,13 @@ providers:
...
+      width: 1223
+      height: 450
```
Comment (sebil) [inline pages/shared/data/tools.yaml]: width and height seems to be just duplicated from the entry above.
Please make sure to have the correct image size as soon as the official logo is also uploaded.

File: pages/shared/data/tools.yaml [hunk 3]
```diff
@@ -316,3 +316,13 @@ providers:
       fit: contain
     formats:
       - email
+  - name: Mailgun
+    url: https://mailgun.com/
```
Comment (sebil) [inline pages/shared/data/tools.yaml]: The official redirected url for the site is https://www.mailgun.com/

**Comment by matthiasrohmer** [comment]
Also adding @pbakaus for explicit approval of the tool.

File: pages/shared/data/tools.yaml (hunk 1 above)
Comment (NLaff) [reply pages/shared/data/tools.yaml]: Hi Sebil, thanks for your feedback. I have uploaded the official logo to that path in my branch. I also updated the dimensions to match the uploaded image, and updated the url.

**Comment by NLaff** [comment]
Thanks Sebil, I believe I've resolved all of your issues.

Here is the relevant blog post Mailgun put out when they started supporting amp email: https://www.mailgun.com/blog/mailgun-supports-amp-email

**Comment by sebil** [comment]
Thanks Nick, looks good to me!
Let's wait for @pbakaus to approve the tool, then we can merge.

**Comment by pbakaus** [comment]
Approved!

**Pull request merged**